src/RequestList.tsx

```diff
@@ -29,6 +33,7 @@ function RequestList({ requests, selectedRequest, selectRequest} : { requests: R
       <TableHead>
         <TableCell>URL</TableCell>
         <TableCell>Capabilities</TableCell>
+        <TableCell>Round Trip Time</TableCell>
```

Comment:
Maybe use "RTT" wrapped in an abbreviation element?
```suggestion
        <TableCell><abbr title="Round Trip Time">RTT</abbr></TableCell>
```